data/categories.txt

```diff
@@ -1444,7 +1444,7 @@ fr:Magasin de vins et spiritueux|vins et spiritueux|4alcool
 de:5Spirituosengeschäft|Spirituosen
 hu:Italbolt|ital
 id:Toko alkohol|minuman keras
-it:Enoteca
+it:Negozio di liquori|Negozio di alcolici|Negozio di vini|Enoteca
```

Comment:
Do you think that users will be able to guess and type phrases in the search exactly as you typed them here?

Reply:
Yes sir, in fact that's how I found this edit was necessary. 
https://context.reverso.net/translation/english-italian/liquor+store
I also added "Negozio di vini" (which is wine shop) because 99% of the time liquor shops also sell wine (I've never been in a liquor store that doesn't sell wine, at least in Italy and Sweden).

Reply:
> Then it may make sense to add hints, to display suggestions after 5 characters for example.

I don't know about that, because if you type "negoz" there will be also suggestions for other kind of shops, for instance electronics shop (negozio di elettronica) or clothes shop (negozio di abbigliamento). But maybe it would be ok to have hints for Enoteca, maybe after 4 or even 3 characters, since I think there are not much other words starting with "eno".

Reply:
Yes, adding a hint to eno would be great

Reply:
> Yes, adding a hint to eno would be great

Done!